config/node1.toml

```diff
@@ -30,7 +30,7 @@ password = ["config/password"]
 
 [mining]
 force_sealing = true
-min_gas_price = 1000000000
+min_gas_price = 0
```

Comment:
Arbitrary accounts of the network shouldn't have an ability to use zero gas price. What's the reason of zero on this line?

Reply:
To let some account use zero gas price, you can call [`Certifier.certify()`](https://github.com/poanetwork/pos-contracts/blob/924b61525eec1ee38afa4a6ec8d48850343dbadd/contracts/Certifier.sol#L32) and [`TxPermission.addAllowedSender()`](https://github.com/poanetwork/pos-contracts/blob/924b61525eec1ee38afa4a6ec8d48850343dbadd/contracts/TxPermission.sol#L29) by the owner. The owner is added into these contracts by default on genesis, so he can have zero balance and use zero gas price to call those functions.

Reply:
Without it I unfortunately get a `truffle migrate` error:
```
Error: Returned values aren't valid, did it run Out of Gas?
```

Reply:
@varasev, I didn't try it. It works now, thanks!